crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -175,6 +182,110 @@ impl PgManager {
             .await
     }
 
+    async fn multi_get_coins(
+        &self,
+        address: Vec<u8>,
+        coin_type: Option<String>,
+        first: Option<u64>,
+        after: Option<String>,
+        last: Option<u64>,
+        before: Option<String>,
+    ) -> Result<Option<(Vec<StoredObject>, bool)>, Error> {
+        let mut query = objects::dsl::objects.into_boxed();
+        query = query
+            .filter(objects::dsl::owner_id.eq(address))
+            .filter(objects::dsl::owner_type.eq(1));
+
+        if let Some(coin_type) = coin_type {
+            query = query.filter(objects::dsl::coin_type.eq(coin_type));
+        }
+
+        if let Some(after) = after {
+            let after = self.parse_obj_cursor(&after)?;
+            query = query
+                .filter(objects::dsl::checkpoint_sequence_number.gt(after))
+                .order(objects::dsl::checkpoint_sequence_number.asc());
+        } else if let Some(before) = before {
+            let before = self.parse_obj_cursor(&before)?;
+            query = query
+                .filter(objects::dsl::checkpoint_sequence_number.lt(before))
+                .order(objects::dsl::checkpoint_sequence_number.desc());
+        }
+
+        let limit = first.or(last).unwrap_or(10) as i64;
```

Comment:
is 10 the default number we agreed upon? Is it somewhere documented?

Reply:
Jsonrpc defaults to 50, idk why I picked 10. We can have this as a method on PgManager and return 50